src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -260,6 +260,31 @@ public void SslStream_StreamToStream_Write_ReadByte_Success()
 
         [OuterLoop] // TODO: Issue #11345
         [Fact]
+        public async Task SslStream_StreamToStream_WriteAsync_ReadByte_Success()
+        {
+            VirtualNetwork network = new VirtualNetwork();
```

Comment:
Would likely have the issue again, if the inner stream didn't implement WriteAsync or ReadAsync